lua/weapons/weapon_dodgeball/cl_init.lua

```diff
@@ -5,7 +5,21 @@ SWEP.Instructions = "Primary: Throw hard\nSecondary: Throw soft\nReload: Pass"
 SWEP.DrawAmmo = false
 SWEP.ViewModelFOV = 85
 
+function SWEP:ThrownBallExists()
+    local ply = self:GetOwner()
+    if(IsValid(self:GetThrownBall()))then return true end
+    if(IsValid(self.fakethrownball))then return true end
+    
+    local lookent = ply:GetEyeTrace().Entity
```

Comment:
So if they look at a dodgeball across the room the one they are holding dissapears?

Reply:
This is meant to be a quick way of getting the ball you threw as soon as its transmitted to you, since thrownball can be networked late, and that creates a small period of time where two dodgeballs are onscreen at once. I guess I could have it ignore balls with a  networked owner other than the player.

Reply:
Probably just do a range check

Reply:
The owner doesnt get unset when its sitting on the ground